src/game/client/in_main.cpp

```diff
@@ -498,7 +574,11 @@ void IN_Attack2Down( const CCommand &args ) { KeyDown(&in_attack2, args[1] );}
 void IN_Attack2Up( const CCommand &args ) {KeyUp(&in_attack2, args[1] );}
 void IN_UseDown ( const CCommand &args ) {KeyDown(&in_use, args[1] );}
 void IN_UseUp ( const CCommand &args ) {KeyUp(&in_use, args[1] );}
+#ifdef NEO
+void IN_JumpDown(const CCommand& args) { KeyDown(&in_jump, args[1]); }
+#else
 void IN_JumpDown ( const CCommand &args ) {KeyDown(&in_jump, args[1] );}
+#endif // NEO
```

Comment:
Nit: the NEO path and the default path are identical, so we could remove this ifdef guard